### ISSUE TITLE
Views with relationships no longer building

### ISSUE THREAD

**Issue by tylerhubert** [likec4/likec4#884]
Hello!

First off, this diagraming tool is amazing and I look forward to continuing to use it.  I recently tried upgrading from 0.6.0 to 1.4.0 and things went well.  I then noticed the upgrade to 1.6.1 and tried it, but things suddenly started not working because my views disappeared from the build.  It took me a while to track it down, but it has something to do with including relationships and happened in the upgrade from 1.4.0 to 1.5.0.
Startup when running `npm run dev` which is just `likec4 serve` on version 1.5.0:
![image](https://github.com/user-attachments/assets/272c8e94-1419-4fcf-86b2-cfc60a2cf3f4)

Startup on version 1.4.0:
![image](https://github.com/user-attachments/assets/0f1a118c-44a2-47d7-9078-8fa12c8be0d6)

This is the same code base, just changed the installed version of LikeC4.  

In a simplified version of this more complex diagram, I found that the view gets removed when I include a relationship between elements, but shows back up when I remove the relationship.
I'm not sure if I'm doing something wrong, or if Windows isn't a supported environment or something because the VS Code extension can open the view no problem, but when doing the `likec4 serve` or `likec4 build`, the c4:lsp module is reporting that it can't find the view even though it finds all the source files.

Any help is appreciated, or if there is a place I can try and replicate this, I'd be happy to.  Thank you!

**Comment by davydkov** [comment]
Hi @tylerhubert 

Thank you for reporting, and I'm super curious to find out the cause.
Can you try to replicate here https://playground.likec4.dev/w/blank/ and post the model here?
Like, the model and view works in vscode/playground, but not in CLI.

**Comment by davydkov** [comment]
> if Windows isn't a supported environment 

Oh, you have multiple files with views, don't you?
Because playground assumes single document. and if you have multiple files and can't reproduce with playground, I guess Windows (actually directory separator) indeed may cause the issue.

**Comment by tylerhubert** [comment]
I will try and replicate in the playground and get back to you, but yes, I do have multiple files with views in many of them to try and keep the views and the systems they represent close together.  I also have directories and sub-directories that have .c4 files in them, so when they all get put together something is causing problems, but ONLY if I include relationships.  If there are no relationships, things work fine, but that kind of defeats the purpose of the diagrams to not have them.

**Comment by tylerhubert** [comment]
`specification {
  element actor {
    style {
      shape person
      color secondary
    }
  }
  element system
}

model {
  system physicalTop 'Physical Topology' {
        system blah 'blah'
        system blah2 'blah 2'

        blah -> blah2 'blah'
  }
  system logicalTop 'Logical Topologies'
}
views {
    view index{
        // all top level things and their relationships
        include *
    }
    view blah of physicalTop {
        include *
    }
}`

The above model seems to work in the playground, but does NOT work on my Windows machine through the CLI.  It does work through the VS Code Plugin preview functionality though.

**Comment by tylerhubert** [comment]
@davydkov I noticed the updates to v1.8.0, so decided to try going up to there.  I'm now getting different output on the `likec4 serve` command as follows:
![image](https://github.com/user-attachments/assets/57aa39ab-35cf-47dd-9142-666e75bfba33)
I did add a manual view layout to one of my views, so maybe this output is from that for the first part.  But the e.relations.toSorted is not a function seems to be a problem somehow.  Not sure if this helps in tracking down the problem or not.
